src/Symfony/Component/Security/Core/composer.json

```diff
@@ -33,7 +33,8 @@
         "symfony/http-foundation": "",
         "symfony/validator": "For using the user password constraint",
         "symfony/expression-language": "For using the expression voter",
-        "ircmaxell/password-compat": "For using the BCrypt password encoder in PHP <5.5"
+        "ircmaxell/password-compat": "For using the BCrypt password encoder in PHP <5.5",
+        "paragonie/random_compat": "For secure number generation"
```

Comment:
I think, we should document that this library is only a compatibility layer for PHP 5: `"For secure random number generation in PHP 5.x"`